sdk/communication/azure-communication-email/src/main/java/com/azure/communication/email/implementation/EmailsImpl.java

```diff
@@ -49,7 +49,7 @@ public final class EmailsImpl {
 
     /**
      * Initializes an instance of EmailsImpl.
-     *
+     * 
```

Comment:
nit: can these cosmetic space changes be avoided for the "doc" comments in this file?

Reply:
I went through and cleaned up trailing whitespace in this pr.